auth/util/src/test/java/org/wildfly/security/auth/util/RegexNameRewriterTest.java

```diff
@@ -0,0 +1,32 @@
+package org.wildfly.security.auth.util;
+
+import org.junit.Test;
+import org.wildfly.security.auth.util.MutableNameRewriter;
+import org.wildfly.security.auth.util.RegexNameRewriter;
+
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertEquals;
+
+public class RegexNameRewriterTest {
+
+    private static final String BEFORE_REPLACE_USER_NAME = ".user.";
+    private static final String AFTER_REPLACE_ALL_USER_NAME = "dotuserdot";
+    private static final String AFTER_REPLACE_USER_NAME = "dotuser.";
+
+    @Test
+    public void testRegexNameRewriter() {
+        Pattern pattern = Pattern.compile("\\.");
+
+        RegexNameRewriter regexNameRewriter = new RegexNameRewriter(pattern, "dot", true);
+        MutableNameRewriter rewriter = new MutableNameRewriter(regexNameRewriter);
```

Comment:
@mittalaishwarya You don't need to use MutableNameRewriter at all to test RegexNameRewriter. You can just call method rewriteName on the regexNameRewriter